src/bot.coffee

```diff
@@ -237,6 +237,7 @@ class SlackBot extends Adapter
     newUser =
       id: user.id
       name: user.name
+      display_name: user.profile.display_name
```

Comment:
sorry, expanding the user API to include `display_name` is not something we are open to doing at this time.